### ISSUE TITLE
Is this project still alive ?

### ISSUE THREAD

**Issue by brunetton** [rockiger/awiki#1]
Hi, I'm wondering whether this project is still alive as there's no README, and I've seen that you started a (new ?) implementation using React [here](https://github.com/rockiger/awiki-react)

**Comment by rockiger** [comment]
No, it is alive anymore. The react version works pretty well for me, but won't be further developed either.

I am working right now on a web based project, that saves all your wiki files in Google drive. Check it out if you want: https://awiki-245320.appspot.com
